yt_dlp/extractor/vrt.py

```diff
@@ -89,24 +90,36 @@ def _extract_formats_and_subtitles(self, data, video_id):
 
     def _call_api(self, video_id, client='null', id_token=None, version='v2'):
         player_info = {'exp': (round(time.time(), 3) + 900), **self._PLAYER_INFO}
-        player_token = self._download_json(
-            'https://media-services-public.vrt.be/vualto-video-aggregator-web/rest/external/v2/tokens',
-            video_id, 'Downloading player token', headers={
+        vrt_player_token = self._download_json(
+            f'https://media-services-public.vrt.be/vualto-video-aggregator-web/rest/external/{version}/tokens',
+            video_id,
+            'Downloading player token',
+            'Failed to download player token',
+            headers={
                 **self.geo_verification_headers(),
                 'Content-Type': 'application/json',
-            }, data=json.dumps({
-                'identityToken': id_token or {},
-                'playerInfo': jwt_encode_hs256(player_info, self._JWT_SIGNING_KEY, headers={
-                    'kid': self._JWT_KEY_ID,
-                }).decode(),
-            }, separators=(',', ':')).encode())['vrtPlayerToken']
+            },
+            data=json.dumps({
+                'identityToken': id_token or '',
+                'playerInfo': jwt_encode_hs256(
+                    player_info,
+                    self._JWT_SIGNING_KEY,
+                    headers={'kid': self._JWT_KEY_ID},
+                ).decode(),
+            }, separators=(',', ':')).encode(),
+        )['vrtPlayerToken']
 
         return self._download_json(
             f'https://media-services-public.vrt.be/media-aggregator/{version}/media-items/{video_id}',
```

Comment:
@bergoid's version used `f'https://media-services-public.vrt.be/vualto-video-aggregator-web/rest/external/{version}/videos/{video_id}',`, which seems to be the correct call that's made when looking at the Network tab

```suggestion
            f'https://media-services-public.vrt.be/vualto-video-aggregator-web/rest/external/{version}/videos/{video_id}',
```

Reply:
The one I proposed is just the actual call from the Network tab

Reply:
I personally think it would be more to use `https://media-services-public.vrt.be/vualto-video-aggregator-web/rest/external/v2/videos/totaly_not_real_video_id` as that's the endpoint people see in the Network tab of their browser.

Reply:
Well you see both, but sure doesn't really matter.